src/test/scala/mesosphere/marathon/integration/EventSubscribersIntegrationTest.scala

```diff
@@ -17,7 +22,7 @@ class EventSubscribersIntegrationTest
     marathon.subscribe("http://localhost:1337").code should be(200)
 
     Then("a notification should be sent to all the subscribers")
-    waitForEvent("subscribe_event").info should contain("callbackUrl" -> "http://localhost:1337")
+    eventually { waitForEvent("subscribe_event").info should contain("callbackUrl" -> "http://localhost:1337") }
```

Comment:
why eventually?
waitForEvent is doing the same as eventually - it actively waits for an event to arrive. 
You basically extend the timeout, or do I miss something?